R/legend.R

```diff
@@ -103,6 +103,7 @@ draw_legend <- function(gsplot) {
         legend <- lapply(legend, function(x) {unname(sapply(x, function(x) {eval(x)}))})
         # clean out arguments not allowed by legend 
         legend <- legend[na.omit(match(names(default.args), names(legend)))]
+        par(usr=view.usr)
```

Comment:
no I do not, but when we had the print order with `draw_axis` and then `draw_custom_grid`, the `usr` remained these values throughout `draw_legend`. So actually, the better way to handle this would be to change `usr` in `print`. Then I wouldn't need to alter `draw_legend` at all